### PR TITLE
feature/CS-666

### PR DESCRIPTION
**Pull request by SuneetaDevi**
Removed duplicate address from contact section

NO errors
NO warnings
Website is working fine

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -762,14 +762,21 @@ <h2>Contact info</h2>
                                 <li>
                                     <span class="icon bi bi-geo-alt"></span>
                                     <strong>Address, </strong>
-                                    10 capreol capreol court, Toronto
+                                    <a href="https://goo.gl/maps/cVfifCcR3DCtpKju6" target="_blank">10 capreol court,
```
Comment (FaizanKhalidMohsin) [inline index.html:765]: Capreol Court, Should be Capitalized.

Comment (FaizanKhalidMohsin) [reply index.html:765]: The address should be exactly: 10 Capreol Court, Toronto, M5V 4B3

**Review by FaizanKhalidMohsin — CHANGES_REQUESTED** [review]
The address should be exactly: 10 Capreol Court, Toronto, M5V 4B3

**Review by SuneetaDevi — COMMENTED** [review]
Done with changes

**Review by FaizanKhalidMohsin — APPROVED** [review]

**Pull request merged**